[Merged by Bors] - Mitigate Test_multipleCPs.. timeout

piersy opened the pull request:
## Motivation
The `Test_multipleCPs..` tests seem to be timing out on develop, I believe this is as a result of removing the `SharedRoundClock` in #4121.

## Changes
Increased the timeouts for these tests.

## Test Plan
See that CI passes

## DevOps Notes
<!-- Please uncheck these items as applicable to make DevOps aware of changes that may affect releases -->
- [x] This PR does not require configuration changes (e.g., environment variables, GitHub secrets, VM resources)
- [x] This PR does not affect public APIs
- [x] This PR does not rely on a new version of external services (PoET, elasticsearch, etc.)
- [x] This PR does not make changes to log messages (which monitoring infrastructure may rely on)

piersy committed:
Increase network delay to try to mitigate timeout

Test_multipleCPs was timing out, increasing the network delay increases
the round times and should hopefully make the test more reliable.

piersy committed:
Increase overall timeots for Test_multipleCPs...

countvonzero commented:
> I believe this is as a result of removing the SharedRoundClock in https://github.com/spacemeshos/go-spacemesh/pull/4121.

how do you determine this? 
https://github.com/spacemeshos/go-spacemesh/issues/4001 already documented such timeout.

piersy commented:
@countvonzero 

The test is now failing fairly regularly whereas before I had not witnessed it failing when running tests with `-p 1`. See these recent jobs:
https://github.com/spacemeshos/go-spacemesh/actions/runs/4388352031/jobs/7684748852
https://github.com/spacemeshos/go-spacemesh/actions/runs/4387428276/jobs/7682715479
https://github.com/spacemeshos/go-spacemesh/actions/runs/4387765368/jobs/7683451294

https://github.com/spacemeshos/go-spacemesh/pull/4121 is the most recent PR to change that test.

countvonzero commented on hare/flows_test.go:
```diff
@@ -434,7 +435,8 @@ func Test_multipleCPsAndIterations(t *testing.T) {
 
-	test.WaitForTimedTermination(t, 100*time.Second)
+	// There are 5 rounds per layer and totoalCPs layers and we double for good measure.
+	test.WaitForTimedTermination(t, 2*networkDelay*5*time.Duration(totalCp))
```
shouldn't this be longer if we expect one of the round will fail, hence causing the 2nd iteration?

piersy replied:
Yes I think it should, would we want to double it again then, to account for each consensus process running twice?

piersy replied:
Actually I see that its just one layer that runs more than a single iteration, so I think we can just increase the timeout for one more layer.

piersy committed:
Increase timeout for Test_multipleCPsAndIterations

Since the test accounts for a stalled layer the timeout needed to
account for that.

Also fixed a typo

piersy commented:
bors merge
